include/s3select_oper.h

```diff
@@ -1534,52 +1534,47 @@ class logical_operand : public base_statement
   }
   virtual value& eval_internal()
   {
-    bool res;
```

Comment:
@galsalomon66 Hi Gal, another thing i think we can improve is the lexical scope of `res`. if you think we should keep it here for simplicity. i will close this PR. otherwise i will revise this change by moving the `return` back to each of the branches.

Reply:
i will look into that. (lexical scope of res)

Reply:
thanks. nothing serious here. but i think the smaller the better regarding the lexical scope.

Reply:
clang-tidy warning #71